photon-lib/src/main/java/org/photonvision/PhotonCamera.java

```diff
@@ -43,9 +43,17 @@
 
 /** Represents a camera that is connected to PhotonVision. */
 public class PhotonCamera {
-    protected final NetworkTable rootTable;
+    protected final NetworkTable cameraTable;
+
+    // NetworkTables entries
+    // Used
     RawSubscriber rawBytesEntry;
     BooleanEntry driverModeEntry;
+    StringSubscriber versionEntry;
+    BooleanPublisher inputSaveImgEntry, outputSaveImgEntry;
+    IntegerEntry pipelineIndexEntry, ledModeEntry;
+    IntegerSubscriber heartbeatEntry;
+
     BooleanPublisher driverModePublisher;
```

Comment:
What are these publishers, subscribers, and entires for? They are uninitiated and unused by any file as far as I can tell.

Reply:
heartbeatEntry, pipelineIndexEntry, ledModeEntry, inputSaveImgEntry/outputSaveImgEntry and versionentry are all used....

Reply:
Yes, im referring to those bellow, the ones I tagged with `// unused`